includes/Core/Util/Auto_Updates.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Class Google\Site_Kit\Core\Util\Auto_Updates
+ *
+ * @package   Google\Site_Kit\Core\Util
+ * @copyright 2022 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Util;
+
+/**
+ * Utility class for auto-updates settings.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class Auto_Updates {
+	/**
+	 * Checks whether plugin auto-updates are enabled for the site.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return bool `false` if auto-updates are disabled, `true` otherwise.
+	 */
+	public static function is_plugin_autoupdates_enabled() {
+		if ( false === self::is_sitekit_autoupdates_forced() ) {
+			return false;
+		}
+
+		if ( function_exists( 'wp_is_auto_update_enabled_for_type' ) ) {
+			return wp_is_auto_update_enabled_for_type( 'plugin' );
+		}
+
+		return false;
+	}
+
+	/**
+	 * Check whether the site has auto updates enabled for Site Kit.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return bool `true` if auto updates are enabled, otherwise `false`.
+	 */
+	public static function is_sitekit_autoupdates_enabled() {
+		if ( true === self::is_sitekit_autoupdates_forced() ) {
+			return true;
+		}
```

Comment:
Shouldn't we return `false` here if the updates are forced disabled as well, eg:

```suggestion
		if ( true === self::is_sitekit_autoupdates_forced() ) {
			return true;
		}
		
		if ( false === self::is_sitekit_autoupdates_forced() ) {
			return false;
		}
```

Reply:
Yes.